<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditBoxDialog</class>
 <widget class="QDialog" name="CreditBoxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cr√©dit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelPhoto">
     <property name="text">
      <string>Image :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelgithub">
     <property name="text">
      <string>git hub :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelSHA1">
     <property name="text">
      <string>SHA1 :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelCompileDate">
     <property name="text">
      <string>Date : </string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelCompileTime">
     <property name="text">
      <string>Time : </string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelcredit">
     <property name="text">
      <string>Credit label</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButtonFermer">
     <property name="text">
      <string>Fermer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
